sdk/core/azure-core/vcpkg.json

```diff
@@ -1,69 +1,73 @@
 {
-    "name": "azure-core-cpp",
-    "version-string": "1.4.0-beta.1",
-    "dependencies": [
+  "name": "azure-core-cpp",
+  "version-string": "1.4.0-beta.1",
+  "dependencies": [
+    {
+      "name": "openssl",
```

Comment:
Please put this alphabetically - easier on diff, and vcpkg ti will be asier when doing manual vcpkg debugging - vcpkg auto-formats stuff, and it will put the package alphabetically, but every time there will be a change that is bigger than necessary.